wallboxapi.js

```diff
@@ -19,6 +19,7 @@ wallboxAPI.prototype={
 					url: `/emails/${email}`,
 					headers: {
 						'Content-Type': 'application/json',
+						'User-Agent': PluginName
```

Comment:
@valiquette would be awesome if we could also get the plugin version here to give more visibility on the other side in case any bug it's detected and fixed here

do you think this is something we can do?

Reply:
yea, sounds reasonable.

Reply:
merged then updated to include version